00-RELEASENOTES

```diff
@@ -1,3 +1,53 @@
+Redis 6.2 RC3   Released Tue Feb 1 14:00:00 IST 2021
+================================================================================
+
+Upgrade urgency LOW: This is the third Release Candidate of Redis 6.2.
+
+Here is a comprehensive list of changes in this release compared to 6.2 RC2,
+each one includes the PR number that added it, so you can get more details
+at https://github.com/redis/redis/pull/<number>
+
+New commands / args:
```

Comment:
```suggestion
New commands and arguments:
```

Reply:
we used `args` in the last release. and we have several references to `arg` below in this release.
i think we can stick to it (and also avoid the extra push to the PR and the long CI ti'll trigger)

Reply:
Sure